src/lib/y2storage/dialogs/callbacks/activate_luks.rb

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env ruby
+#
+# encoding: utf-8
+
+# Copyright (c) [2017] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "ui/dialog"
+require "y2storage/secret_attributes"
+
+Yast.import "UI"
+
+module Y2Storage
+  module Dialogs
+    module Callbacks
+      # Dialog to manage luks activation callbacks
+      class ActivateLuks < UI::Dialog
+        include SecretAttributes
+
+        secret_attr :encryption_password
+
+        def initialize(uuid, attempt)
```

Comment:
Once you start telling the user number of failed attempts, they will start asking for `How much do I have left?` and start feeling the urge, and start feeling the discomfort. We don't want our users to feel bad about good things (we even want them to feel good about bad ones ;)).